src/main/target/serial_post.h

```diff
@@ -54,7 +54,11 @@ Configuration used:
 #include "common/utils.h" // BIT, LOG2
 
 /****                                UART                                 *****/
```

Comment:
src/utils/gen-serial.py must be modified and this file regenerated (see comments in this and gen-serial file)

Reply:
One of the reasons why I don't like the use of generator scripts from the initial generation. This is a simple example.

Wait until we come to the io pins when a port has 32, not 16.

Reply:
@blckmn : It depends. The generator was used so that only template is maintained and all long, repetitive code is generated, even after major changes. Generated code is long and 'flat', not intended for manual editing.

16->32 it almost trivial with IO.h code. Just modify few lines.